### PR TITLE
Update HTML Sanitizer doc for `max_input_length` option

### PR DESCRIPTION
**Pull request by tristan-lg**
Add missing documentation for https://symfony.com/doc/current/html_sanitizer.html#allow-elements

Related to the bug https://github.com/symfony/symfony/issues/49040

### CODE REVIEW

**Comment by OskarStark** [comment]
friendly ping @tgalopin

**Comment by tristan-lg** [comment]
Do you know why the "CI/Code Blocks" fails ?

**Review by tgalopin — APPROVED** [review]
You could also explain why if you want (it's because it's a source of DoS attacks)

File: html_sanitizer.rst [hunk 1]
```diff
@@ -931,6 +931,76 @@ the HTML sanitizer: ``src``, ``href``, ``lowsrc``, ``background`` and ``ping``.
                 ->allowRelativeMedias()
         );
 
+Configure Max input length
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To prevent DoS attacks, the HTML sanitizer limits the input length to ``20000`` by default.
+Using this option, you can change the max input length.
+Inputs longer than this value will be truncated.
```
Comment (OskarStark) [inline html_sanitizer.rst]:
```suggestion
Using this option, you can change the max input length, Inputs longer than this value will be truncated.
```

File: html_sanitizer.rst [hunk 2]
```diff
@@ -931,6 +931,76 @@ the HTML sanitizer: ``src``, ``href``, ``lowsrc``, ``background`` and ``ping``.
                 ->allowRelativeMedias()
         );
 
+Configure Max input length
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (OskarStark) [inline html_sanitizer.rst]:
```suggestion
Max Input Length
~~~~~~~~~~~~~~~~
```

File: html_sanitizer.rst [hunk 3]
```diff
@@ -931,6 +931,76 @@ the HTML sanitizer: ``src``, ``href``, ``lowsrc``, ``background`` and ``ping``.
...
+        # config/packages/html_sanitizer.yaml
+        framework:
+            html_sanitizer:
+                sanitizers:
+                    app.post_sanitizer:
+                        # ...
+
+                        # specifies the max input length. Inputs longer than this value will be
+                        # truncated (default: 20000)
+                        max_input_length: 20000
```
Comment (OskarStark) [inline html_sanitizer.rst]: If 20000 is the default, we should configure 30k here.

the comment can be removed completely, instead I would suggest to write:

```suggestion
                        max_input_length: 30000 # default: 20000
```

**Comment by javiereguiluz** [comment]
Tristan, thanks and congrats on a great first Symfony Docs contribution 🎉 

Thanks to reviewers too 🙌

**Comment by javiereguiluz** [comment]
Closing manually as merged. GitHub shows it as "closed", but it was merged; see https://github.com/symfony/symfony-docs/commit/e39cbf6dbc6ab2e0c65dcc5ebef120edf78c96fe

**Pull request closed without merging**